javascript/node/selenium-webdriver/test/lib/capabilities_test.js

```diff
@@ -126,4 +127,21 @@ describe('Capabilities', function() {
       assert.deepEqual({bar: 123}, caps[Symbols.serialize]());
     });
   });
+
+  describe('Test StrictFileInteractability capability', function(env){
+    it('Test Uploading With Invisible File Input When StrictFileInteractability Is On', async function(){
```

Comment:
`should fail to upload files to a non interactable input when StrictFileInteractability is on`